src/components/NotificationModal/index.tsx

```diff
@@ -162,6 +158,26 @@ export const NotificationModal: React.FC<NotificationModalProps> = ({
               />
             </ExtraFieldContainer>
           ))}
+        {showAddCriteria && (
+          <>
+            <RegexSelectorInputContainer>
+              {regexSelectorProps.map((props, i) => (
+                <RegexSelectorInput
+                  isVisibleDelete={i !== 0}
+                  dataCyPrefix={i}
+                  {...props}
+                />
+              ))}
+            </RegexSelectorInputContainer>
+            <AddCriteriaContainer
+              data-cy="add-regex-selector-button"
+              onClick={onClickAddRegexSelector}
+            >
+              <Icon glyph="Plus" />
+              Add additional criteria
+            </AddCriteriaContainer>
```

Comment:
It looks like im able to add unlimited criteria boxes even tho theres no other options i can select.
![image](https://user-images.githubusercontent.com/4605522/87700627-a0674180-c764-11ea-87b8-45cd4519bc75.png)

Reply:
i added this functionality and included a test